mycore-base/src/main/java/org/mycore/common/config/MCRConfigurationInputStream.java

```diff
@@ -99,9 +99,7 @@ public static MCRConfigurationInputStream getMyCoRePropertiesInstance() throws I
             LogManager.getLogger().info("Current configuration directory: {}",
                 configurationDirectory.getAbsolutePath());
             // set MCR.basedir, is normally overwritten later
-            if (configurationDirectory.isDirectory()) {
-                initStream = getBaseDirInputStream(configurationDirectory);
-            }
+            initStream = getBaseDirInputStream(configurationDirectory);
```

Comment:
Maybe log a warning about the missing base directory? Could help in cases  other then the first startup where there actually is a problem because of a missing or inaccessible directory.

Reply:
I think the log statement above is sufficient. If an exception occurs later, that should contains all the details, e.g. a specific subdirectory that is not accessible.